assets/scss/modules/_navigation.scss

```diff
@@ -147,3 +145,131 @@ a.back-to-top-link {
   }
 
 }
+
+/*
+Page Navigation
+
+A page navigation element with a previous and next link.
+
+Markup:
+<nav role="navigation" aria-label="Page navigation">
+  <ul class="page-nav">
+    <li class="page-nav__item page-nav__item--previous">
+      <a class="page-nav__link page-nav__link-previous" title="Navigate to previous part" rel="prev" href="#">
+        <span class="page-nav__label">Previous </span>
+        <span class="page-nav__title">Overview</span>
+      </a>
+    </li>
+    <li class="page-nav__item page-nav__item--next">
+      <a class="page-nav__link page-nav__link-next" title="Navigate to next part" rel="next" href="#">
+        <span class="page-nav__label">Next</span>
+        <span class="page-nav__title">Inheritance Tax reliefs</span>
+      </a>
+    </li>
+  </ul>
+</nav>
+
+Styleguide Navigation.Page Navigation
+*/
+
+.page-nav {
+  display:block;
+  margin:4em 0 0 0;
+  padding:0.75em 0;
+  border-bottom:1px solid $light-grey;
+
+  @include contain-floats;
+
+  @include media(mobile) {
+    margin:2em 0 0 0
+  }
+}
+
+.page-nav__item {
+  font-size:16px;
```

Comment:
Can you use the `core-*` mixins from [govuk toolkit](https://github.com/hmrc/assets-frontend/blob/master/assets/govuk_frontend_toolkit/stylesheets/_typography.scss) for font-sizes please?